plutip.cabal

```diff
@@ -191,6 +193,46 @@ test-suite plutip-tests
     RecordWildCards
     TupleSections
 
+executable plutip-server
+  import:           common-language
+  import:           common-ghc-options
+  main-is:          Main.hs
+  hs-source-dirs:   plutip-server
+  default-language: Haskell2010
+  build-depends:
+    , aeson
+    , base
+    , base16-bytestring
+    , bytestring
+    , cardano-api
+    , cardano-wallet
+    , cardano-wallet-launcher
+    , data-default
+    , directory
+    , exceptions
+    , extra
+    , filepath
+    , http-types
+    , mtl
+    , optparse-applicative
+    , plutip
```

Comment:
Just a suggestion for the future: as the plutip-server doesn't seem to actually depend on BPI, I think we should separate `plutip-core` from `tasty-plutip` in the future.